### ISSUE TITLE
[Bug] PopUp looks different on iOS

### ISSUE THREAD

**Issue by alex3326** [xamarin/XamarinCommunityToolkit#1753]
**Description**
Simple popup looks different on Android and iOS devices. The background shadow is on the inside on iOS not on the outside. See images to compare results.

**Basic Information**

IDE: Visual Studio 2019 v.16.11.7
Xamarin.Forms: 5.0.0.2244
Xamarin.CommunityToolkit: 1.3.1
Android TestDevice: Android 11.0
iPhone TestDevice: iOS 15.0.2

**XML Code**
![popUpXml](https://user-images.githubusercontent.com/23524888/144371080-bfc4b338-5015-4686-81cf-7ed6d36a5e9c.PNG)

**Android**
![android](https://user-images.githubusercontent.com/23524888/144371081-3185c5f4-1510-484c-9597-ee359165541b.PNG)

**iOS**
![iOS](https://user-images.githubusercontent.com/23524888/144371083-0c62b393-b377-4175-b2cb-bf5133b2ab12.jpg)

**Comment by bijington** [comment]
@alex3326 are you using Dark mode on the iOS device?

**Comment by alex3326** [comment]
> [bijington](/bijington)

Yes but the dark background is still missing on the light mode. Or is there never a dark background like on Android?

Thanks for the hint ^^.

**Comment by bijington** [comment]
> @bijington 
> 
> 
> 
> Yes but the dark background is still missing on the light mode. Or is there never a dark background like on Android?
> 
> 
> 
> Thanks for the hint ^^. 

Sorry I don't know too much around Popups by my initial thoughts are that it is down to the appearance set on the phone.

The below shows the same popup being rendered on the same device with just changing the appearance setting on iOS.

There should be a similar setting on Android, have you got that set to light or dark?

Light mode
![image](https://user-images.githubusercontent.com/17139988/144478526-bbb31c84-779c-49e8-96d6-6acc30de6df3.png)

Dark mode
![image](https://user-images.githubusercontent.com/17139988/144478558-29eb2aa0-08fb-4506-8929-333d4eda9761.png) 


Anyway if you want to override the colours you see here you can set the `BackgroundColor` or `Color` of the popup or inner controls to what you would like.

**Comment by nacompllo** [comment]
Exactly the same thing happens to me in iOS, the dark background does not appear, is it a bug? or is it the expected behavior in iOS? On Android, however, it works correctly and shows the background darkened. I have prepared a repository on GitHub to be able to reproduce it :
https://github.com/nacompllo/PopupBackgroundTransparent

If anyone can shed any information on this I would appreciate it.

<img width="759" alt="PopupBackgroundColor" src="https://user-images.githubusercontent.com/20904914/150385936-df09cc9d-8002-41a7-b44c-1d9654b8008c.PNG">

**Comment by karimElJed** [comment]
As normal system popups have a darkened background I would assume this is a bug or at least a missing feature. @VladislavAntonyuk is the behaviour the same for the MAUI version?

**Comment by dasunmax89** [comment]
<img width="579" alt="Screenshot 2023-05-15 at 20 33 00" src="https://github.com/xamarin/XamarinCommunityToolkit/assets/50762737/183a27c8-babf-4f63-80e4-2e11a09394a4">

I did a small workaround by showing a full-screen contentview of BackgroundColor="#CC000000"
